packages/liveblocks-react/src/room.tsx

```diff
@@ -408,6 +408,48 @@ function makeExtrasForClient<M extends BaseMetadata>(client: OpaqueClient) {
     return;
   }
 
+  const loadRoomThreads = (
+    room: Room<JsonObject, LsonObject, BaseUserMeta, Json, M>,
+    queryKey: string,
+    options: UseThreadsOptions<M>
+  ): Promise<ThreadData<M>[]> => {
+    const existingPromise = store.getLoadThreadsPromise(queryKey);
+    if (existingPromise !== undefined) return existingPromise;
+
+    const getThreadsAndInboxNotifications = async () => {
+      const result = await room.getThreads(options);
+
+      const lastRequestedAt = lastRequestedAtByRoom.get(room.id);
+
+      /**
+       * We set the `lastRequestedAt` value for the room to the timestamp returned by the current request if:
+       * 1. The `lastRequestedAt` value for the room has not been set
+       * OR
+       * 2. The `lastRequestedAt` value for the room is older than the timestamp returned by the current request
+       */
+      if (
+        lastRequestedAt === undefined ||
+        lastRequestedAt > result.requestedAt
+      ) {
+        lastRequestedAtByRoom.set(room.id, result.requestedAt);
+      }
+
+      poller.start(POLLING_INTERVAL);
```

Comment:
This is one thing I'm not sure about. Did we change behavior here? In the inbox notifications case, we only start polling once a component using `useInboxNotifications` is mounted, and we stop polling when the last component using it unmounts. Here we're always starting a poller, but never stopping it?

And slight aside: from just this PR diff, it's unclear to me _which_ poller this is starting? Do we have a poller per query for threads? Or do we have a single poller for _all_ threads?